edk2toollib/uefi/edk2/parsers/dsc_parser_test.py

```diff
@@ -0,0 +1,21 @@
+# @file dsc_parser_test.py
```

Comment:
This file seems unrelated...

Reply:
I'm modifying the DSC parser so I think it makes sense to add a test for it :) 
As it stands right now, testing !includes in the parser would be pretty hard without creating a bunch of files. I guess I could do it

Reply:
Added a much better test